test/run_test.sh

```diff
@@ -2,6 +2,12 @@
 
 #DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" >/dev/null && pwd )"
 
-export PYTHONPATH=../lib:$PATH:$PYTHONPATH
+export PYTHONPATH=${PYTHONPATH}:/opt/adios2/lib/python3.5/site-packages
```

Comment:
Did we fix python to be 3.5?

Reply:
This is for adios2 python binding. Currently, it is fixed to python3.5 as I compiled adios2 with that version.